INSTALLATION.md

```diff
@@ -1,29 +1,127 @@
+you plan to use it.
+
+If your goal is to look into the code, modify and improve it, go straight
+to the [source installation](#installing-for-development) section.
+
+Taipy GUI needs your system to have Python 3.8 or above installed.
+
+## Installing from PyPI
+
+The easiest way to install Taipy GUI is from the
+[Pypi software repository](https://pypi.org/project/taipy-gui/).
+
+Run the command:
+```console
+$ pip install taipy-gui
 ```
-pip install taipy-gui
+
+If you are running in a virtual environment, you will have to
+issue the command:
+```console
+$ pipenv install taipy-gui
 ```
 
-## Development version
+These commands install the `taipy-gui` package in the Python environment
+with all its dependencies.
 
-You can install the development version of _taipy_ with _pip_ and _git_:
-```
-pip install git+https://git@github.com/Avaiga/taipy-gui
+## Installing from GitHub
+
+The development version of Taipy GUI is updated daily with changes from the
+Taipy R&D and external contributors that we praise for their input.
+
+This development version of Taipy GUI can be installed using _pip_ and _git_:
+```console
+$ pip install git+https://git@github.com/Avaiga/taipy-gui
 ```
 
-## Work with the _taipy-gui_ code
+## Installing for development
+
+If you need the source code for Taipy GUI on your system, so you can see
+how things are done or maybe participate in the improvement of the package,
+you can clone the GitHub repository:
+
+```console
+$ git clone https://github.com/Avaiga/taipy-gui.git
 ```
-git clone https://github.com/Avaiga/taipy-gui.git
-cd taipy-gui
-pip install .
+
+This creates the 'taipy-gui' directory that holds all the package's source code.
+
+### Building the JavaScript bundle
+
+Taipy GUI has some code dealing with the client side of the Web applications.
+This code is written in [TypeScript](https://www.typescriptlang.org/), relies on
+[React](https://reactjs.org/) components, and is packaged into a JavaScript bundle
+that is sent to browsers when they connect to all Taipy GUI applications.
+
+Here is how you can build this bundle.
+
+You need to make sure that the [Node.js](https://nodejs.org/) JavaScript runtime version 18
+or above is installed on your machine.<br/>
+Note that Node.js comes with the [`npm` package manager](https://www.npmjs.com/) as part
+of the standard installation.
+
+Here is the sequence of commands that must be issued, assuming your current directory
+is the root directory of the repository:
+
+```console
+# Current directory is the repository's root directory
+# Move to the 'gui' directory
+$ cd gui
+# Install the DOM dependencies (once and for all)
+$ cd dom
+$ npm i
+$ cd ..
+# Install the Web app dependencies
+$ npm i --omit=optional
+# Build the Web application
+$ npm run build
+$ cd ..
+# Current directory is the repository's root directory
 ```
 
-If you want to run tests, please install `Pipenv`:
+This creates the directory `src/taipy/gui/webapp` in the root directory of the repository,
```

Comment:
```suggestion
This creates the directory `src/taipy/gui/webapp` in the root directory of the repository
```

Reply:
This is exactly the kind of remarks I would have done... Thx!